test/integration/auth.spec.ts

```diff
@@ -332,35 +335,52 @@ describe('admin.auth', () => {
 
         // Login to set the lastRefreshTime.
         await firebase.auth!().signInWithEmailAndPassword('lastRefreshTimeUser@example.com', 'p4ssword')
-          .then(() => admin.auth().getUser('lastRefreshTimeUser'))
-          .then((userRecord) => {
-            expect(userRecord.metadata.lastRefreshTime).to.exist;
-            expect(isUTCString(userRecord.metadata.lastRefreshTime!));
-            const creationTime = new Date(userRecord.metadata.creationTime).getTime();
-            const lastRefreshTime = new Date(userRecord.metadata.lastRefreshTime!).getTime();
+          .then(async () => {
```

Comment:
This was changed in one of the PRs upstream, the `minimal-auth` branch simply isn't updated (will fix)

Reply:
Fixed